builtin/providers/aws/resource_aws_route53_record.go

```diff
@@ -270,6 +270,9 @@ func resourceAwsRoute53RecordCreate(d *schema.ResourceData, meta interface{}) er
 		zone, *rec.Name, req)
 
 	respRaw, err := changeRoute53RecordSet(conn, req)
+	if err != nil {
+		return err
```

Comment:
Good idea, are you thinking a simple `return fmt.Errorf("[ERR]: Error building changeset: %s", err)` or something involving `errwrap`?